hail/python/hail/genetics/reference_genome.py

```diff
@@ -80,6 +80,21 @@ def par_tuple(p):
                                [par_tuple(p) for p in config['par']],
                                _builtin)
 
+
+    @classmethod
+    def _from_backend_config(cls, config, _builtin=False):
+        def par_tuple(p):
+            assert p['start']['contig'] == p['end']['contig']
```

Comment:
This shouldn't be user-facing, ever, since we're controlling the correctness of this assertion from both ends.

(Actually, this was mostly copy-pasted from `_from_config` because I thought I needed a different function, but it turns out I was just serializing incorrectly so I deleted this again.)